.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: tests
+
+on:
+  pull_request:
+    types: ['opened', 'synchronize']
+    paths-ignore:
+      - 'README.md'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'README.md'
+jobs:
+  unit:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+    steps:
+      -
```

Comment:
The same yaml formatting comment above (and throughout the rest down below)

Reply:
done